pages/dao/[symbol]/proposal/[pk].tsx

```diff
@@ -28,28 +33,76 @@ const Proposal = () => {
     relativeNoVotes,
     relativeYesVotes,
   } = useProposalVotes(proposal?.info)
+  const { realmInfo } = useRealm()
+  const wallet = useWalletStore((s) => s.current)
+  const signatories = useWalletStore((s) => s.selectedProposal.signatories)
+  const connection = useWalletStore((s) => s.connection)
+
+  const [showSignOffModal, setShowSignOffModal] = useState(false)
```

Comment:
Let's put SignOff button into the right hand action panel together with the other proposal action buttons (Approve, Deny, Cancel etc..)